src/test/scripts/org/reaktivity/k3po/nukleus/ext/simplex/server.sent.throttle.initial.only.update.none/client.rpt

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "nukleus://server/streams/client"
+        option nukleus:route 1234L
+        option nukleus:transmission "simplex"
+        option nukleus:window 8192
+connected
+
+write "Hello"
+write ", "
```

Comment:
Please read my comment on the other script before reading this one. Where I suggest you do in this script is as follows:
```
write "H"
write await FIRST_BYTE_RECEIVED
write "e"
write aborted
```
Combined with the changes I suggested to the other script the test should pass. Given that it is currently passing I would expect that it will probably fail because you won't see the "write aborted". Once you have made the test changes and confirmed the test fails then I can work on fixing it so the test passes.

Reply:
I've updated the scripts, both client and server as you requested and the test fails. I've filed an issue for it: https://github.com/kaazing/tickets/issues/1206 and ignored the test in this PR untul it will be fixed.